bot/cogs/search.py

```diff
@@ -201,68 +201,144 @@ async def anime(self, ctx: Context, *, query: str) -> None:
+                    url = f"https://kitsu.io/manga/{manga_id}"
+
+                    embed = Embed(title=f"{title}", color=ctx.author.color, url=url)
+                    embed.description = manga["attributes"]["synopsis"][0:425] + "..."
+
+                    if manga["attributes"]["averageRating"]:
+                        embed.add_field(name="Average Rating", value=manga["attributes"]["averageRating"])
+                    embed.add_field(name="Popularity Rank", value=manga["attributes"]["popularityRank"])
+
+                    if manga["attributes"]["ageRating"]:
+                        embed.add_field(name="Age Rating", value=manga["attributes"]["ageRating"])
+                    embed.add_field(name="Status", value=manga["attributes"]["status"])
+                    thing = "" if not manga["attributes"]["endDate"] else f' to {manga["attributes"]["endDate"]}'
+                    embed.add_field(name="Published", value=f"{manga['attributes']['startDate']}{thing}")
+
+                    if manga["attributes"]["chapterCount"]:
+                        embed.add_field(name="Chapters", value=manga["attributes"]["chapterCount"])
+                    embed.add_field(name="Type", value=manga["attributes"]["mangaType"])
+                    embed.set_thumbnail(url=manga["attributes"]["posterImage"]["original"])
+
+                    try:
+                        await ctx.send(f"**{title}** - <{url}>", embed=embed)
+                    except Exception:
+                        aired = f"{manga['attributes']['startDate']}{thing}"
+                        template = textwrap.dedent(
+                            f"""
+                            ```
+                            url: {url}
+                            Title: {title}
+                            Average Rating: {manga["attributes"]["averageRating"]}
+                            Popularity Rank: {manga["attributes"]["popularityRank"]}
+                            Age Rating: {manga["attributes"]["ageRating"]}
+                            Status: {manga["attributes"]["status"]}
+                            Aired: {aired}
+                            Type: {manga['attributes']["showType"]}
+                            Powered by HotWired
+                            ```
+                            """
+                        )
+                        await ctx.send(template)
+
+    @command()
+    async def weather(self, ctx: Context, *, city: str = None) -> None:
+        """Sends current weather in the given city name
+
+        eg. weather london"""
+        url_formatted_city = city.replace(" ", "-")
+        async with aiohttp.ClientSession() as session:
+            try:
+                weather_lookup_url = f"https://api.openweathermap.org/data/2.5/weather?q={url_formatted_city}&appid={config.WEATHER_API_KEY}"
+                async with session.get(weather_lookup_url) as resp:
+                    data = await resp.json()
+            except Exception:
```

Comment:
Same problem as in suggestion above (don't except `Exception` until you really know what you're doing)